www-servers/pound/pound-3.0.2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+MY_P="${P/p/P}"
+
+DESCRIPTION="A http/https reverse-proxy and load-balancer"
+HOMEPAGE="https://www.apsis.ch/pound.html"
+SRC_URI="https://www.apsis.ch/pound/${MY_P}.tgz"
+
+LICENSE="BSD GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~x86"
+
+DEPEND="
+	dev-libs/libpcre:=
+	dev-libs/libyaml:=
+	dev-libs/nanomsg:=
+	dev-libs/openssl:=
+	net-libs/mbedtls:=
+	elibc_musl? ( sys-libs/queue-standalone )
+"
+
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+DOCS=( README.md )
+
+PATCHES=(
+	"${FILESDIR}"/${P}-hpack.patch
+)
+
+src_install() {
+	rm GPL.txt || die
```

Comment:
You're not calling cmake_src_install. Is there a reason for that?

Reply:
Using cmake_src_install results in `ninja: error: unknown target 'install'`, so I suppose there is no install target defined.

Reply:
Okay!